modules/op-executing-pipeline-run-and-task-run-with-custom-scc-and-service-account.adoc

```diff
@@ -0,0 +1,91 @@
+[id=op-executing-pipeline-run-and-task-run-with-custom-scc-and-service-account_{context}]
+= Executing pipeline run and task run with custom SCC and service account
+
+When using the `pipelines-scc` security context contraint (SCC) associated with the default `pipelines` service account (SA), the pipeline run and task run pods may face timeouts. This happens because in the default `pipelines-scc` SCC, the `fsGroup.type` parameter is set to `MustRunAs`.
+
+To avoid pod timeouts, you can create a custom SCC with the `fsGroup.type` parameter set to `RunAsAny`, and associate it with a custom service account.
+
+[NOTE]
+====
+As a best practice, use custom SCC and SA for pipeline runs and task runs. This approach allows greater flexibility and does not break the runs, when the defaults are modified during an upgrade.
+====
+
+[dicrete]
```

Comment:
I don't think you need `[dicrete]` for .these types of headings. I think they don't show up in the TOC anyway?